src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -85,6 +86,7 @@ public function configure(array $configuration = null)
 
         $this->regex = '/^'.$indent.' \* @(?:'.implode('|', $types).')'.$desc.'\s*$/u';
         $this->regexCommentLine = '/^'.$indent.' \*(?! @)(?:\s+(?P<desc>\V+))(?<!\*\/)$/u';
+        $this->collapse = (bool) $this->configuration['collapse'];
```

Comment:
casting is pointless, for non-boolean values `FixerConfigurationResolver` will raise exception

Reply:
fixed, cleaned CS, reran builds)